src/Solvers/sparse_approximate_inverse.jl

```diff
@@ -13,53 +13,64 @@ mutable struct SpaiIterator{VF<:AbstractVector, SV<:SparseVector, VI<:AbstractVe
 end
 
 """
-the sparse_approximate_inverse function calculates a SParse Approximate Inverse M ≈ A⁻¹ to be used as a preconditioner
-Since it can be applied to the residual with just a matrix multiplication instead of the solution
-of a triangular linear problem it makes it very appealing to GPU use
+    sparse_approximate_inverse(A::AbstractMatrix; ε::Float64, nzrel)
 
-The algorithm implemeted here calculates M following the specifications found in
+Compute a Sparse Approximate Inverse `M ≈ A⁻¹` to be used as a preconditioner.
+Since it can be applied to the residual with just a matrix multiplication instead
+of the solution of a triangular linear problem it makes it very appealing to use
+on the GPU.
 
-Grote M. J. & Huckle T, "Parallel Preconditioning with sparse approximate inverses" 
+The algorithm implemeted here computes `M` following the specifications found in
+
+> Grote M. J. & Huckle T, "Parallel Preconditioning with sparse approximate inverses" 
 
 In particular, the algorithm tries to minimize the Frobenius norm of
 
-‖ A mⱼ - eⱼ ‖ where mⱼ and eⱼ are the jₜₕ column of matrix M and the identity matrix I, respectively
+```
+‖ A mⱼ - eⱼ ‖
+```
 
-Since we are solving for an "sparse approximate" inverse (i.e. a sparse version of the actually non-sparse A⁻¹),
-we start assuming that mⱼ has a sparsity pattern J, which means that
+where `mⱼ` and `eⱼ` denote the `jₜₕ` column of matrix `M` and the identity
+matrix `I`, respectively. Since we are solving for an "sparse approximate" inverse
+(i.e., a sparse version of the actually non-sparse `A⁻¹`), we start assuming that
```

Comment:
Should this clarification come at the top?

Reply:
Good point. Moved it and rephrased a bit.